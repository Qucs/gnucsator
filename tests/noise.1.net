# Qucs

Vnoise:N0 _net0 gnd u=1
Vnoise:N1 _net1 _net0 u=1
Vnoise:N2 _net2 gnd u=2
Vnoise:N3 _net3 gnd u=1 e=2
.AC:AC1 Type="lin" Start="1 MHz" Stop="2 MHz" Points="3" Noise="yes"
