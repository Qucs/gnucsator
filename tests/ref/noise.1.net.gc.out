<Qucs Dataset>
default plugins: ignored
.AC:AC1 Type="l ... ="2 MHz" Points="3" Noise="yes" 
                                        ^ ? what's this (incomplete)?
Gnucap   System status
iterations: op=3, dc=0, tran=0, fourier=0, total=6
transient timesteps: accepted=0, rejected=0, total=0
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=4, subckt=0, model=4, total=8
dctran density=65.6%, ac density=65.6%

#Freq                      vr(_net0)                 vr(_net1)                 vr(_net2)                 vr(_net3)                 vi(_net0)                 vi(_net1)                 vi(_net2)                 vi(_net3)                
 1.E+6                     0.                        0.                        0.                        0.                        0.                        0.                        0.                        0.                       
 1.5E+6                    0.                        0.                        0.                        0.                        0.                        0.                        0.                        0.                       
 2.E+6                     0.                        0.                        0.                        0.                        0.                        0.                        0.                        0.                       
